### ISSUE TITLE
make it so that .bashrc is sourced before jupyter notebook and sage worksheets are started

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/smc#341]
Don't just do it before starting the local_hub server, since a slight mistake could make project not accessible.

Requested by: Mani Chandra

**Comment by williamstein** [comment]
Definitely still an issue...  Also `~/.bash_profile` is not sourced.  These are only sourced if a bash terminal is run.

That makes sense, actually.  Closing this in favor of @timothyclemansinsea's https://github.com/sagemathinc/smc/issues/845
